pkg/core/native/native_neo.go

```diff
@@ -246,11 +246,11 @@ func (n *NEO) updateCommittee(ic *interop.Context) error {
 		return ic.DAO.PutStorageItem(n.ContractID, prefixCommittee, si)
 	}
 
-	committee, cvs, err := n.computeCommitteeMembers(ic.Chain, ic.DAO)
+	_, cvs, err := n.computeCommitteeMembers(ic.Chain, ic.DAO)
 	if err != nil {
 		return err
 	} else if cvs == nil {
```

Comment:
But do we really need this check then? If `computeCommitteeMembers` guarantees that `committee` and `cvs` and non-nil in case of `err == nil` it can just be dropped.

Reply:
Fixed.